docs/docusaurus.config.ts

```diff
@@ -35,60 +35,77 @@ const ingoPreset = {
 const navBarLeftSide = [
   {
     position: 'left',
-    label: 'Docs',
+    label: 'Guide',
     to: '/',
-  }
-] satisfies NavbarItem[]
-
-const navBarRightSide = [
+    activeBaseRegex: '^/$', // exact match for home
```

Comment:
This ONLY matches the home route `/`. Better to have it active on anything other than api:

```suggestion
    activeBaseRegex: '^(?!\/api).*', // everything not starting with /api
```